### ISSUE TITLE
Error with summariseCharacteristics when running in Snowflake

### ISSUE THREAD

**Issue by cbarbozaerasmus** [darwin-eu-dev/PatientProfiles#390]
Hey, we are getting an error when running summariseCharacteristics() in Snowflake. We don't encounter this error in other dbms like Postgresql. 

```
# Monotherapy with ERAS
cli::cli_alert_info("Instantiating Treatments Cohorts")
cohortSetTreatments <- readCohortSet(here::here("cohorts", "pah_treatments"))
cdm <- generateCohortSet(cdm, cohortSetTreatments, name = "pah_treatments", overwrite = TRUE)

# Treatments summaries
cli::cli_alert_info("Summarising Treatments")
summaryTreatments <- cdm$pah_cohort_treatment %>%
  summariseCharacteristics(
    strata =  list(c("age_group"),
                   c("sex")),
    tableIntersect = list(),
    cohortIntersect = list(
      "Treatments" = list(
        targetCohortTable = "pah_treatments",
        value = "flag",
        window = list(c(0, 0),
                      c(1, 30),
                      c(1, 90),
                      c(1, 365))
      )),
    minCellCount = 5
  )
```

```
ℹ Instantiating Treatments Cohorts
ℹ Summarising Treatments
→ C1-003-Combination-Therapy has been changed to c1_003_combination_therapy
→ C1-003-PDE-5is has been changed to c1_003_pde_5is
→ C1-003-Other-PAH-Therapy has been changed to c1_003_other_pah_therapy
→ C1-003-ERAs has been changed to c1_003_eras
→ some provided names were not in snake_case
→ names have been changed to lower case
→ special symbols in names have been changed to '_'
Error: nanodbc/nanodbc.cpp:1769: 00000: String '1_to_30' is too long and would be truncated 
<SQL> 'INSERT INTO "STUDY_REFERENCE"."wfhac_win_count_flag" SELECT
  "subject_id",
  "index_date",
  "id_name",
  "window_name",
  CASE WHEN ("count" > 0.0) THEN 1.0 WHEN NOT ("count" > 0.0) THEN 0.0 END AS "flag"
FROM (
  SELECT *, '1_to_30' AS "window_name"
  FROM (
    SELECT "subject_id", "index_date", "count", "id_name"
    FROM (
      SELECT "subject_id", "index_date", "id", SUM("indicator") AS "count"
      FROM "STUDY_REFERENCE"."wfhac_window"
      GROUP BY "subject_id", "index_date", "id"
    ) "LHS"
    LEFT JOIN "dbplyr_059"
      ON ("LHS"."id" = "dbplyr_059"."id")
  ) "q01"
) "q02"'
```

**Comment by catalamarti** [comment]
Hi @cbarbozaerasmus  how are you connected to the database? we found some errors last week related to DatabaseConnector.
Are you connecting using DBI?

**Comment by cbarbozaerasmus** [comment]
Yes, we are connecting through DBI.

**Comment by catalamarti** [comment]
Can you try to run with only one window? @cbarbozaerasmus

**Comment by catalamarti** [comment]
In another study it was reported same issue and they managed to run with a different call for each window, definitely worth to investigate but at least we have a "solution" to run it short term

**Comment by cbarbozaerasmus** [comment]
We'll try that.

**Comment by catalamarti** [comment]
hi @ginberg do you have access to the iqvia test snowflake? can you have a call at some point to try to debug this? It seems that it is not study specific. Maybe we can create a reprex and then try to fix.
Thank youu!

**Comment by ginberg** [comment]
@catalamarti yes I have access to the iqvia snowflake db, happy to work on a reprex for this

**Comment by catalamarti** [comment]
@cbarbozaerasmus this should produce same result and work, meanwhile I will try to work with @ginberg to create a reprex
```
summaryTreatments <- cdm$pah_cohort_treatment %>%
  summariseCharacteristics(
    strata =  list(c("age_group"),
                   c("sex")),
    tableIntersect = list(),
    cohortIntersect = list(
      "Treatments" = list(targetCohortTable = "pah_treatments", value = "flag", window = c(0, 0)),
      "Treatments" = list(targetCohortTable = "pah_treatments", value = "flag", window = c(1, 30)),
      "Treatments" = list(targetCohortTable = "pah_treatments", value = "flag", window = c(1, 90)),
      "Treatments" = list(targetCohortTable = "pah_treatments", value = "flag", window = c(1, 365))
    ),
    minCellCount = 5
  )
```

**Comment by catalamarti** [comment]
solved in #396
